library/src/scala/IArray.scala

```diff
@@ -273,6 +273,51 @@ object opaques:
     extension [T](arr: IArray[T]) def zip[U: ClassTag](that: IArray[U]): IArray[(T, U)] =
       genericArrayOps(arr).zip(that)
   }
+
+  /** Conversion from IArray to immutable.ArraySeq */
+  given genericWrapIArray[T]: Conversion[IArray[T], scala.collection.immutable.ArraySeq[T]] =
```

Comment:
Based on https://github.com/scala/scala/blob/2.13.x/src/library/scala/Predef.scala#L544

Not sure if this is the best location for this conversion. We could also move it to Predef.

Reply:
Strange, it seems that implicit conversions have higher priority than extension methods. @odersky, is that the intended behaviour?

Reply:
I had to add a DummyImplicit to make it work